src/signing/networkMessages/kusama.js

```diff
@@ -82,7 +82,10 @@ export async function UnstakeTx(
     const validatorAddresses = delegatedValidators.filter(
       (validator) => !from.includes(validator)
     )
-    transactions.push(await api.tx.staking.nominate(validatorAddresses))
+    // transaction will fail if targets are empty
```

Comment:
that is bad: that would mean you can not reduce nominations to none

Reply:
to do that you need to to `forceUnstake`

Reply:
https://polkadot.js.org/api/substrate/extrinsics.html#forceunstake-stash-t-accountid-num-slashing-spans-u32

Maybe there is some other command. @mariopino do you know?

I can confirm nominate fails when validators addresses is `[]`

Reply:
[Here ](https://polkascan.io/pre/kusama/transaction/0xc4204d60b987f0aa93e3ec7a11cfcbc1d711338865dc27f77735a73be9dfce0c)the failed transaction

Reply:
This `api.tx.staking.chill` maybe can do the job:
https://polkadot.js.org/api/substrate/extrinsics.html#chill

It does the same thing as `forceUnstake` but effects will be felt at the beginning of the next era.

Reply:
chill man

Reply:
let's add the chill logic?